R/method-differentialAbundance.R

```diff
@@ -4,8 +4,8 @@
 #' 
 #' @param data AbsoluteAbundanceData object
 #' @param comparisonVariable string identifying the metadata column to be used when grouping samples.
-#' @param groupA array of strings, each string indicating a value in the comparisonVariable column. Samples with these metadata values will be included in Group A for the differential abundance calculation. Must have no overlap with groupB
-#' @param groupB array of strings, each string indicating a value in the comparisonVariable column. Samples with these metadata values will be included in Group B for the differential abundance calculation. Must have no overlap with groupA.
+#' @param groupA array of strings or a BinList, with each string indicating a value in the comparisonVariable column or each bin a different range for a continuous variable. Samples with these metadata values will be included in Group A for the differential abundance calculation. Must have no overlap with groupB
```

Comment:
why a string or BinList ?? everything besides the bin label is optional iirc, so you can use it for the categorical vars as well. we do that elsewhere for the map..